share/lib/python/neuron/expect_hocerr.py

```diff
@@ -20,19 +20,23 @@ def expect_hocerr(callable, args, sec=None):
   old_stderr = sys.stderr
   sys.stderr = my_stderr = StringIO()
   err = 0
+  pyerrmes = False
   try:
     if sec:
       callable(*args, sec=sec)
     else:  
       callable(*args)
   except:
+    pyerrmes = sys.exc_info()
     err=1
   errmes =  my_stderr.getvalue()
   sys.stderr = old_stderr
   if errmes:
     errmes = errmes.splitlines()[0]
     errmes = errmes[(errmes.find(':')+2):]
     print("expect_hocerr: %s" % errmes)
+  elif pyerrmes:
+    print(str(pyerrmes[0]) + ':' + str(pyerrmes[1]))
   if err == 0:
     print("expect_hocerr: no err for %s%s" % (str(callable), str(args)))
   assert(err)
```

Comment:
`assert` is a statement, not a function: ` assert condition, message`. This notation works because `assert(err)` is evaluated as `assert bool(err)` but can lead to defunct assert statements when someone comes along and modifies this as `assert(err, message)` which will always evaluate to `True` since now it's evaluated as `assert bool((err, message))` and a non empty tuple is truthy.